Resources/ServerInfo/Guidebook/Science/Robotics.xml

```diff
@@ -7,7 +7,7 @@
 <GuideEntityEmbed Entity="ExosuitFabricator" Caption="Exosuit Fabricator"/>
 </Box>
   The [color=#a4885c]Exosuit Fabricator[/color] is where you'll be printing out most of the parts for your creations, such as:
-  - Bot limbs
+  - Cyborg limbs
```

Comment:
i recommend linking the deeper Cyborg article in the Robotics section of this one, since they are closely related.

Reply:
I'll do that. Good point.